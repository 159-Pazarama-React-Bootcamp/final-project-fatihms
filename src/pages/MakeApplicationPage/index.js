import React from "react";

import ApplicationForm from "../../containers/ApplicationForm";
import Header from "../../components/Header";

import styles from "./styles.module.css";

function MakeApplicationPage() {
  return (
    <div className={styles["make-application-container"]}>
      <header>
        <Header />
      </header>
      <main>
        <article>
          <ApplicationForm />
        </article>
      </main>
    </div>
  );
}

export default MakeApplicationPage;
