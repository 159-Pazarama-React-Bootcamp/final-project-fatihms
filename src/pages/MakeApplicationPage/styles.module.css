.mac-link {
  color: white;
  text-decoration: none;
  display: flex;
  align-items: center;
  justify-content: center;
  background-color: #f05454;
  padding: 8px 16px;
  border-radius: 4px;
}

.mac-link > i {
  margin-left: 10px;
}

main {
  height: 90vh;
  display: flex;
  flex-direction: column;
  justify-content: center;
}
