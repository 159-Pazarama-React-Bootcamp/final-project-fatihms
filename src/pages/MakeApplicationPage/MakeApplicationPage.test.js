/*eslint-disable */

import React from "react";

import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";
import { MemoryRouter as Router } from "react-router-dom";
import userEvent from "@testing-library/user-event";
import "@testing-library/jest-dom/extend-expect";

import MakeApplicationPage from "./index";

describe("MakeApplicationPage", () => {
  beforeEach(() => {
    render(
      <Router>
        <MakeApplicationPage />
      </Router>
    );
  });
  test("MakeApplicationPage renders correctly", () => {
    const tree = renderer
      .create(
        <Router>
          <MakeApplicationPage />
        </Router>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
