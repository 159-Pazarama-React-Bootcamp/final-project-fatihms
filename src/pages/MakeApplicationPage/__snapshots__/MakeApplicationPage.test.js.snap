// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MakeApplicationPage MakeApplicationPage renders correctly 1`] = `
<div
  className="make-application-container"
>
  <header>
    <div
      className="header-container"
    >
      <h2>
        Başvuru Oluştur
      </h2>
      <a
        className="mac-link"
        href="/basvuru-sorgula"
        onClick={[Function]}
      >
        <h3>
          Başvuru Sorgula
        </h3>
        <i>
          <svg
            fill="currentColor"
            height={30}
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 1024 1024"
            width={30}
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M909.6 854.5L649.9 594.8C690.2 542.7 712 479 712 412c0-80.2-31.3-155.4-87.9-212.1-56.6-56.7-132-87.9-212.1-87.9s-155.5 31.3-212.1 87.9C143.2 256.5 112 331.8 112 412c0 80.1 31.3 155.5 87.9 212.1C256.5 680.8 331.8 712 412 712c67 0 130.6-21.8 182.7-62l259.7 259.6a8.2 8.2 0 0 0 11.6 0l43.6-43.5a8.2 8.2 0 0 0 0-11.6zM570.4 570.4C528 612.7 471.8 636 412 636s-116-23.3-158.4-65.6C211.3 528 188 471.8 188 412s23.3-116.1 65.6-158.4C296 211.3 352.2 188 412 188s116.1 23.2 158.4 65.6S636 352.2 636 412s-23.3 116.1-65.6 158.4z"
            />
          </svg>
        </i>
      </a>
    </div>
  </header>
  <main>
    <article>
      <section
        className="application-form"
      >
        <div
          className="form-container form-container--55"
        >
          <form
            className="af-form"
            onSubmit={[Function]}
          >
            <div
              className="form-container-left"
            >
              <h4
                className="oneRow"
              >
                Kişisel
              </h4>
              <div
                className="input-container"
              >
                <input
                  id="firstName"
                  name="firstName"
                  onChange={[Function]}
                  placeholder="Fatih Mustafa"
                  type="text"
                  value=""
                />
                <label
                  htmlFor="firstName"
                >
                  Ad
                </label>
              </div>
              <div
                className="input-container"
              >
                <input
                  id="lastName"
                  name="lastName"
                  onChange={[Function]}
                  placeholder="Sağır"
                  type="text"
                  value=""
                />
                <label
                  htmlFor="lastName"
                >
                  Soyad
                </label>
              </div>
              <div
                className="input-container"
              >
                <input
                  id="age"
                  name="age"
                  onChange={[Function]}
                  placeholder="25"
                  type="number"
                  value=""
                />
                <label
                  htmlFor="age"
                >
                  Yaş
                </label>
              </div>
              <div
                className="oneRow"
              >
                <div
                  className="input-container"
                >
                  <input
                    id="tc"
                    name="tc"
                    onChange={[Function]}
                    placeholder="25215151515"
                    type="text"
                    value=""
                  />
                  <label
                    htmlFor="tc"
                  >
                    TC Kimlik No
                  </label>
                </div>
              </div>
              <h4
                className="oneRow"
              >
                İletişim
              </h4>
              <div
                className="oneRow"
              >
                <div
                  className="input-container"
                >
                  <input
                    id="address"
                    name="address"
                    onChange={[Function]}
                    placeholder="Yeşil sok. No:1 / 34"
                    type="text"
                    value=""
                  />
                  <label
                    htmlFor="address"
                  >
                    Adres
                  </label>
                </div>
              </div>
              <div
                className="input-container"
              >
                <input
                  id="city"
                  name="city"
                  onChange={[Function]}
                  placeholder="İstanbul"
                  type="text"
                  value=""
                />
                <label
                  htmlFor="city"
                >
                  Şehir
                </label>
              </div>
              <div
                className="input-container"
              >
                <input
                  id="district"
                  name="district"
                  onChange={[Function]}
                  placeholder="Beşiktaş"
                  type="text"
                  value=""
                />
                <label
                  htmlFor="district"
                >
                  İlçe
                </label>
              </div>
            </div>
            <span
              className="form-container-hr"
            />
            <div
              className="form-container-right"
            >
              <h4
                className="oneRow"
              >
                Başvuru
              </h4>
              <div
                className="oneRow"
              >
                <div
                  className="input-container"
                >
                  <input
                    id="reason"
                    name="reason"
                    onChange={[Function]}
                    placeholder="Sorun"
                    type="text"
                    value=""
                  />
                  <label
                    htmlFor="reason"
                  >
                    Başvuru Nedeni
                  </label>
                </div>
              </div>
              <div
                className="oneRow"
              >
                <label
                  className="sc-bdvvtL iciELI  "
                  htmlFor="file"
                >
                  <input
                    accept=".jpg,.png,.gif,.docx"
                    name="file"
                    onChange={[Function]}
                    type="file"
                  />
                  <svg
                    fill="none"
                    height="32"
                    viewBox="0 0 32 32"
                    width="32"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M5.33317 6.66667H22.6665V16H25.3332V6.66667C25.3332 5.196 24.1372 4 22.6665 4H5.33317C3.8625 4 2.6665 5.196 2.6665 6.66667V22.6667C2.6665 24.1373 3.8625 25.3333 5.33317 25.3333H15.9998V22.6667H5.33317V6.66667Z"
                      fill="#0658C2"
                    />
                    <path
                      d="M10.6665 14.6667L6.6665 20H21.3332L15.9998 12L11.9998 17.3333L10.6665 14.6667Z"
                      fill="#0658C2"
                    />
                    <path
                      d="M25.3332 18.6667H22.6665V22.6667H18.6665V25.3333H22.6665V29.3333H25.3332V25.3333H29.3332V22.6667H25.3332V18.6667Z"
                      fill="#0658C2"
                    />
                  </svg>
                  <div
                    className="sc-dkPtRN ebdQkY"
                  >
                    <span
                      className="sc-hKwDye kCudqJ"
                    >
                      <span>
                        Upload
                      </span>
                       or drop a file right here
                    </span>
                    <span
                      className="file-types"
                      title="types: JPG,PNG,GIF,docx"
                    >
                      JPG,PNG,GIF,docx
                    </span>
                  </div>
                </label>
                <p
                  style={
                    Object {
                      "marginTop": "10px",
                    }
                  }
                >
                  no files uploaded yet
                </p>
              </div>
              <button
                className="button-item"
                type="submit"
              >
                Gönder
              </button>
            </div>
          </form>
        </div>
      </section>
    </article>
    <p>
      Eğer bir sorun yaşarsanız, lütfen 
      <a
        href="/iletisim"
        onClick={[Function]}
      >
        bize ulaşın.
      </a>
    </p>
  </main>
</div>
`;
